src/components/adjudications/AdjudicationsSection.jsx

```diff
@@ -33,41 +29,30 @@ class AdjudicationsSection extends React.Component {
         };
         adjudications.push(adjudication);
       });
-      this.setState({ adjudications });
+      this.setState({ adjudications, loading: false });
     });
   }
 
   render() {
     const { match: { params: { eventId, performanceId }}} = this.props;
-    const { adjudications } = this.state;
+    const { adjudications, loading } = this.state;
     const collectionName = `events/${eventId}/performances/${performanceId}/adjudications`;
     const headings = ['Judge', 'Audio', 'Cumulative Score', 'Awards'];
     const showAdjudications = Array.isArray(adjudications) && adjudications.length > 0;
 
     return (
-      <React.Fragment>
-        <SectionHeader title="adjudication" showNew={false} />
-        <Table>
-          <TableHeader headings={headings} />
-          <TableBody>
-            {showAdjudications && adjudications.map((rowProps) => {
-              const keys = ['artisticMark', 'audio', 'choreoAward', 'cumulativeMark', 'notes', 'judgeName', 'specialAward', 'technicalMark'];
-              const currentValues = pick(rowProps, keys);
-              return (
-                <AdjudicationTableRow
-                  collectionName={collectionName}
-                  currentValues={currentValues}
-                  key={rowProps.id}
-                  id={rowProps.id} />);
-            })}
-          </TableBody>
-        </Table>
-        {!showAdjudications && (
-          <div style={{ display: 'flex', justifyContent: 'center' }}>
-            <EmptyState type="adjudication" title="Empty Adjudications Page" subtitle="Create your first Adjudication" />
-          </div>
-        )}
-      </React.Fragment>
+      <Section headings={headings} loading={loading} showContent={showAdjudications} type="adjudication">
+        {showAdjudications && adjudications.map((rowProps) => {
+          const keys = ['artisticMark', 'audio', 'choreoAward', 'cumulativeMark', 'notes', 'judgeName', 'specialAward', 'technicalMark'];
```

Comment:
If these are consts, can we bring the declaration outside this map? Let's avoid unnecessary declarations in iterations.